parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -284,4 +286,6 @@ echo $(date),$(hostname), endcustomscript >>/opt/m
 mkdir -p /opt/azure/containers && touch /opt/azure/containers/provision.complete
```

Comment:
Not blocking but it's wierd that we're racing reboot in 1 minute here. Shoudl we just write out compelete before rebootrequired if/lese block? we're not checking anything there anyways. Guessing there's a reason though I'm just not aware of. But you could move writing out provision.compete to before validation since we're not reallly checking output of anything in the rebootrequiredblock.

Reply:
ah you and jack started this conversation up above. good.

Reply:
O.K. now I understand why we can't move the check down below (because we can't be doing anything serially after a possible shutdown -r command).